medusa/providers/torrent/json/torrentday.py

```diff
@@ -138,6 +135,10 @@ def parse(self, data, mode):
                                   " minimum seeders: {0}. Seeders: {1}", title, seeders)
                     continue
 
+                # Check if this is a freeleech torrent and if we've configured to only allow freeleech.
+                if self.freeleech and row['download-multiplier'] != 0:
+                    continue
```

Comment:
@p0psicles why not discard freeleech earlier before setting `title` ?  `title = re.sub(...` - line 120